effects/internal/despill.frag

```diff
@@ -7,11 +7,15 @@ uniform sampler2D tex;
 varying vec2 uTexCoord;
 uniform vec2 resolution;
 uniform int channel;
+uniform float factor;
 
 const vec2 renderScale = vec2(1,1);
 
 void main(void)
 {
+    
+    factor = factor * 0.01; // convert factor from 0-100 scale to 0-1 scale
```

Comment:
Factor is a read only variable, you need to make a new `float factorOut` and use that from then on:

```
float factorOut;
factorOut = factor * 0.01;
```
Then replace all the other factors with factorOut

Reply:
I see. Thanks! Fixed!